### ISSUE TITLE
Coordinates and Footprint WKT for sampling-event datasets

### ISSUE THREAD

**Issue by ManonGros** [gbif/portal-feedback#2166]
Both Footprint WKT and coordinates can be included occurrence and sampling-event datasets.

However, I am confused by a few things when it comes to sampling-events:
* The IPT doesn't allow the coordinates to be mapped in the occurrence file. This is a bit counter-intuitive. I would assume that the event file could contain a footprint corresponding to the area sampled and the occurrence file, the coordinates of the observations in this area. Would it make sense to have the possibility to have coordinates in the occurrence file?

* The footprints aren't displayed at the event level, only at the occurrence level which also seems a bit counter-intuitive since they are entered for each event (not occurrence).

* When only the footprints are given, the occurrences appear to have no coordinates (e.g. has_coordinates=False). This is not exactly true since the the footprints contain coordinates. However, since they are not included in the default download, it makes some sense. It is too bad that no-one benefits from this information.

* When both coordinates and footprint are given at the same time, only the coordinates are displayed. And since the footprints aren't in the default download, the information might be overlooked.

Is there anything we could do to improve the visibility of Footprint WKT in the context of sampling-event datasets?

**Comment by MortenHofft** [comment]
> The footprints aren't displayed at the event level, only at the occurrence level which also seems a bit counter-intuitive since they are entered for each event (not occurrence).

We don't model events, so we have no idea of knowing what the WKT is for the event. So everything on the event pages is taken from the occurrences. Which makes it quite fragile. We could take the WKT of the first occurrence with the event ID, but there is nothing hindering the occurrence from having another wkt than the event (eg a more precise location within the event area). So it is adding more fragile to the page. I assume that was the reasoning for not showing it.

**Comment by MortenHofft** [comment]
> When only the footprints are given, the occurrences appear to have no coordinates (e.g. has_coordinates=False). This is not exactly true since the the footprints contain coordinates. However, since they are not included in the default download, it makes some sense. It is too bad that no-one benefits from this information.

This seems unfair I agree. A way to fix it during publishing could be to add a centroid with an uncertainty? Another approach could be that we interpreted the WKT and added the centroid and uncertainty during ingestion?

**Comment by MortenHofft** [comment]
> When both coordinates and footprint are given at the same time, only the coordinates are displayed. And since the footprints aren't in the default download, the information might be overlooked.

That seems wrong, but I'm also unable to reproduce it. Can you give an example please. Below is one with a WKT and a lat/lon where both is shown. https://www.gbif.org/occurrence/1949673350

**Comment by albenson-usgs** [comment]
I agree this is problematic and for sampling event datasets I have been working with I have sometimes been including the lat/lon in the occurrence file in addition to the event file ([example](https://www1.usgs.gov/obis-usa/ipt/resource?r=usgs_glsc_rvcat_trawl)).

I could also see this being problematic in the work I'm doing with the biologging field. Specifically you might have individual points for a remotely sensed (or other automated method) animal but may want to represent all of the movements of the animal as one event with a WKT but have the individual points available as occurrences for modeling and other analyses.

**Comment by ManonGros** [comment]
> That seems wrong, but I'm also unable to reproduce it. Can you give an example please. Below is one with a WKT and a lat/lon where both is shown. https://www.gbif.org/occurrence/1949673350

My bad, I made a mistake on that one, thanks for correcting me!

**Comment by ManonGros** [comment]
> This seems unfair I agree. A way to fix it during publishing could be to add a centroid with an uncertainty? Another approach could be that we interpreted the WKT and added the centroid and uncertainty during ingestion?

The second option would be probably more intuitive for publishers (at they wouldn't have to calculate the uncertainty themselves). Would this be possible?
